lib/purescript-cst/src/Language/PureScript/AST/Exported.hs

```diff
@@ -134,6 +134,9 @@ isExported (Just exps) (KindDeclaration _ _ n _) = any matches exps
   matches declRef = do
     let refName = declRefName declRef
     TyName n == refName || TyClassName (tyToClassName n) == refName
+isExported (Just exps) (RoleDeclaration RoleDeclarationData{..}) = any matches exps
+  where
+  matches declRef = TyName rdeclIdent == declRefName declRef
 isExported (Just exps) decl = any matches exps
   where
   matches declRef = declName decl == Just (declRefName declRef)
```

Comment:
If `declName` also returned the `RoleDeclaration`'s name, then the above case for `RoleDeclaration`s wouldn't be needed. I'll have to see where else `declName` is used and whether it should include `RoleDeclaration`s or whether my current approach is best.